public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -57,7 +57,7 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
         >
           {scopes && <scopes.Component model={scopes} />}
           <NavToolbarActions dashboard={model} />
-          {!isHomePage && controls && hasControls && (
+          {controls && hasControls && (
```

Comment:
@ivanortegaalba - this is not correct, with this change the default home dashboard gets time controls which should not happen. I think we should simply rely on the home dashboard definition, for some reason the default home dashboard shows the controls. Seems like the `controls.hasControls()` may be invalid implementation?

Reply:
Fixed :) it was a bug with the URL sync. Now it should be fixed